bigflow/build/dist.py

```diff
@@ -85,27 +78,67 @@ def build_image(
         image_dir: Path,
         deployment_config: Path):
     os.mkdir(image_dir)
-    tag = build_docker_image_tag(docker_repository, version)
+    tag = bfc.build_docker_image_tag(docker_repository, version)
     build_docker_image(project_dir, tag)
     try:
         export_docker_image_to_file(tag, image_dir, version)
         shutil.copyfile(deployment_config, image_dir / deployment_config.name)
     finally:
-        remove_docker_image_from_local_registry(tag)
+        bfc.remove_docker_image_from_local_registry(tag)
+
+
+def _hook_pregenerate_sdist(command_cls):
+    """
+    Wraps existing distutils.Command class.
+    Runs 'sdist' and copy resutl into 'build/bf-project.tar.gz'
+    """
+
+    class command(command_cls, distutils.cmd.Command):
```

Comment:
UpperCase?

Reply:
I've refactored the code. Now it generate class with exactly same name.